utils/hints.go

```diff
@@ -244,11 +261,20 @@ func GenerateHints(annotations mapstr.M, container, prefix string) mapstr.M {
 						}
 					}
 				}
+
+				if !found {
+					returnerror = fmt.Errorf("provided hint :%v is not in the supported list", parts[1])
+				} //End of check
+
 			}
 		}
 	}
 
-	return hints
+	if returnerror == nil {
```

Comment:
I don't think you need this condition here, since `returnerror` is nil, then it is the same as returning `hints, returnerror` in the line below

Reply:
I updated this. But you are correct I only return now the list `incorrecthints`